ImportBlocks/ImportBlocks.cs

```diff
@@ -42,57 +33,65 @@ private bool OnExport(string[] args)
                         fs.Seek(sizeof(uint), SeekOrigin.Begin);
                     }
                     else
+                    {
+                        byte[] buffer = new byte[sizeof(uint)];
+                        fs.Read(buffer, 0, buffer.Length);
+                        start = BitConverter.ToUInt32(buffer, 0);
+                        fs.Seek(0, SeekOrigin.Begin);
+                    }
+                }
+                else
+                {
+                    if (writeStart)
                     {
                         fs.Write(BitConverter.GetBytes(start), 0, sizeof(uint));
                     }
-                    if (start <= end)
-                        fs.Write(BitConverter.GetBytes(count), 0, sizeof(uint));
-                    fs.Seek(0, SeekOrigin.End);
-                    for (uint i = start; i <= end; i++)
+                }
+                if (start <= end)
+                    fs.Write(BitConverter.GetBytes(count), 0, sizeof(uint));
+                fs.Seek(0, SeekOrigin.End);
+                for (uint i = start; i <= end; i++)
+                {
+                    using (Snapshot snapshot = Blockchain.Singleton.GetSnapshot())
                     {
-                        using (Snapshot snapshot = Blockchain.Singleton.GetSnapshot())
-                        {
-                            Block block = snapshot.GetBlock(i);
-                            byte[] array = block.ToArray();
-                            fs.Write(BitConverter.GetBytes(array.Length), 0, sizeof(int));
-                            fs.Write(array, 0, array.Length);
-                            Console.SetCursorPosition(0, Console.CursorTop);
-                            Console.Write($"[{i}/{end}]");
-                        }
+                        Block block = snapshot.GetBlock(i);
+                        byte[] array = block.ToArray();
+                        fs.Write(BitConverter.GetBytes(array.Length), 0, sizeof(int));
+                        fs.Write(array, 0, array.Length);
+                        Console.SetCursorPosition(0, Console.CursorTop);
+                        Console.Write($"[{i}/{end}]");
                     }
                 }
             }
+        }
+
+        private bool OnExport(string[] args)
+        {
+            bool writeStart;
+            uint start, count;
+            string path;
+            if (args.Length < 2) return false;
+            if (!string.Equals(args[1], "block", StringComparison.OrdinalIgnoreCase)
+                && !string.Equals(args[1], "blocks", StringComparison.OrdinalIgnoreCase))
```

Comment:
seems duplicated

Reply:
What is this refactor?

Reply:
I told him that both conditions `if/else` was very similar, so i asked him to join this two logics in only one

Reply:
This is not duplicated. I think this is to make `export` support `export blocks` and `export block`